docs/docs/evaluating_datasets.rst

```diff
@@ -49,3 +49,12 @@ The following prints the scores defined in WNLI task (f1_micro, f1_macro, accura
     ('f1_micro', 0.65)
     ('f1_micro_ci_low', 0.4000000000000001)
     ('f1_micro_ci_high', 0.8000000000000002)
+
+
+If you want to evaluate with few templates or few num_demos you can run:
+
+.. code-block:: python
+
+  dataset = load_dataset('unitxt/data', 'card=cards.wnli,template=[templates.classification.multi_class.relation.default,templates.key_val],num_demos=[0,1,3],demos_pool_size=10,max_test_instances=100',trust_remote_code=True)
+
+This will randomly sample from the templates and for each instance assign a random template from the list and run number of demonstration from the list.
```

Comment:
I think showing how the results will look like is important, 
namely, how should one parse the results

Reply:
Can you elaborate, what do you mean by "parse the results"?

Reply:
how can one obtain the results for say the first template and first n_shots

Reply:
Added. Will add an automatic solution later on today.